src/list-modules.ts

```diff
@@ -1,8 +1,7 @@
-import * as glob from 'glob'
-import * as path from 'path'
-import { flatten } from './utils'
-import { BuildResolverOptions, ResolverOptions } from './resolvers'
-import { LifetimeType } from './awilix'
+import { basename, cwd, glob, resolve } from './deps.deno.ts'
```

Comment:
Would be it easier to have a `./deps.ts` which internally does `./deps.deno.ts` so only a single file references deno?

Reply:
Sure, that could be a bit cleaner. Good idea, I'll change it.